PayPal.xcodeproj/xcshareddata/xcschemes/UnitTests.xcscheme

```diff
@@ -34,6 +34,20 @@
                ReferencedContainer = "container:PayPal.xcodeproj">
             </BuildableReference>
          </BuildActionEntry>
+         <BuildActionEntry
+            buildForTesting = "YES"
+            buildForRunning = "YES"
+            buildForProfiling = "NO"
+            buildForArchiving = "NO"
+            buildForAnalyzing = "NO">
+            <BuildableReference
+               BuildableIdentifier = "primary"
+               BlueprintIdentifier = "PayPal::PayPalTests"
+               BuildableName = "PayPalTests.xctest"
+               BlueprintName = "PayPalTests"
+               ReferencedContainer = "container:PayPal.xcodeproj">
+            </BuildableReference>
+         </BuildActionEntry>
```

Comment:
Same question here - what are these changes needed for?

Reply:
Also, I notice xcodeproj changes for `PayPalTests.xctest` and `CardTest.xctest` but not `PaymentsCore.xctest`. Is this intentional?

Reply:
Ah OK. So the build command `xcodebuild -workspace 'PayPal.xcworkspace' -sdk 'iphonesimulator' -configuration 'Debug' -scheme 'UnitTests' -destination 'name=iPhone 11,platform=iOS Simulator' test` doesn't need this set to run properly.

I guess Fastlane uses some specific feature of xcodebuild that requires it to be configured for running then.

Reply:
It does - alternatively, I can wrap your build command within the lane, and revert the project changes to allow the tests to be 'run'. The main benefit of using the baked in action is formatting and test output, but those aren't essential. Is there a reason the test scheme shouldn't be able to be 'run' ?

Reply:
This has been reverted, so we no longer need the 'run' ability for our test target.